sale_order_global_stock_route/models/sale_order.py

```diff
@@ -36,10 +36,8 @@ class SaleOrderLine(models.Model):
 
         @api.onchange("product_id")
         def product_id_change(self):
-            res = super().product_id_change()
```

Comment:
As onchange as been removed and to avoid function name collisions, prefer a name like 'onchange_product_id_global_stock_route()'

Reply:
Thanks for the review
Change done